echopop/extensions/inversion/invert.py

```diff
@@ -0,0 +1,47 @@
+from typing import Any, Callable, Dict, Literal, Union
+
+import pandas as pd
+from numpy.typing import ArrayLike
+
+
+def normalize_scattering_model_parameters(
+    scattering_model_parameters: Dict[str, Any],
+) -> Dict[str, Any]:
+    """
+    Normalize the scattering model parameters
+    """
+    # == model_para_conversion.m
+    pass
+
+
+def Sv_prediction_error(
+    measured_Sv: ArrayLike[float],
+    predicted_Sv: ArrayLike[float],
+):
+    """
+    Compute inverted volumetric backscattering strength ($S[v]$) prediction error
+    """
+    pass
+
+
+def invert_population(
```

Comment:
This `invert_population`  and `Sv_prediction_error` are not strictly part of the "inversion" but are derived computations. Let me read through the rest of doc and see where these might go.